src/main/java/uk/gov/hmcts/ccd/RestTemplateConfiguration.java

```diff
@@ -48,11 +54,21 @@
 
     @Bean(name = "restTemplate")
     public RestTemplate restTemplate() {
+
         final RestTemplate restTemplate = new RestTemplate();
         HttpComponentsClientHttpRequestFactory requestFactory = new HttpComponentsClientHttpRequestFactory(getHttpClient());
         requestFactory.setReadTimeout(readTimeout);
         LOG.info("readTimeout: {}", readTimeout);
         restTemplate.setRequestFactory(requestFactory);
+
+        List<HttpMessageConverter<?>> messageConverters = new ArrayList<>();
+        //Add the Jackson Message converter
+        MappingJackson2HttpMessageConverter converter = new MappingJackson2HttpMessageConverter();
+        // Note: here we are making this converter to process any kind of response,
+        // not only application/*json, which is the default behaviour
+        converter.setSupportedMediaTypes(Collections.singletonList(MediaType.ALL));
+        messageConverters.add(converter);
+        restTemplate.setMessageConverters(messageConverters);
```

Comment:
We are changing the configuration of the default rest template used by all http calls in Data Store. Before we do this, please triple check this configuration you are adding is suitable for all the calls and won't cause problems/regressions. 
If there are doubts about it, as an alternative to minimise impact, we can introduce a new method here that creates a rest template dedicated to the `DocumentManagementRestClient`. And apply this configuration only there. And inject that in the `DocumentManagementRestClient`